static/app/utils/replays/replayReader.tsx

```diff
@@ -195,7 +195,13 @@ export default class ReplayReader {
 
   getDomNodes = memoize(() =>
     extractDomNodes({
-      frames: this.getDOMFrames(),
+      frames: this.getDOMFrames().filter(
+        frame =>
+          !(
+            (frame as SlowClickFrame).category === 'ui.slowClickDetected' &&
+            !isDeadClick(frame as SlowClickFrame)
+          )
+      ),
```

Comment:
Can we put this inside of `getDOMFrames`
That way the list of frames (getDOMFrames) and the list of Node (getDomNodes) are both based on the same filtered array